projects/plugins/boost/app/lib/critical-css/Generator.php

```diff
@@ -63,21 +63,4 @@ public function display_generate_meta() {
 		<meta name="<?php echo esc_attr( self::GENERATE_QUERY_ACTION ); ?>" content="true"/>
 		<?php
 	}
-
-	/**
-	 * Add the critical css generation flag to a list if it's present in the URL.
-	 * This is mainly used by filters for compatibility.
-	 *
-	 * @var $query_args    array The list to add the arg to.
-	 *
-	 * @return $query_args array The updatest list with query args.
-	 */
-	public static function add_generate_query_action_to_list( $query_args ) {
```

Comment:
This is still [referenced in `Critical_CSS`](https://github.com/Automattic/jetpack/blob/1d819a29859e18bc7ae444361d2e6f35e3c177b7/projects/plugins/boost/app/modules/optimizations/critical-css/Critical_CSS.php#L51).

Reply:
Ops! Updated.